pkg/fieldpath/fieldpath.go

```diff
@@ -84,15 +84,15 @@ func ExtractFieldPathAsString(obj interface{}, fieldPath string) (string, error)
 
 // SplitMaybeSubscriptedPath checks whether the specified fieldPath is
 // subscripted, and
-//  - if yes, this function splits the fieldPath into path and subscript, and
-//    returns (path, subscript, true).
-//  - if no, this function returns (fieldPath, "", false).
+//   - if yes, this function splits the fieldPath into path and subscript, and
+//     returns (path, subscript, true).
+//   - if no, this function returns (fieldPath, "", false).
 //
 // Example inputs and outputs:
-//  - "metadata.annotations['myKey']" --> ("metadata.annotations", "myKey", true)
-//  - "metadata.annotations['a[b]c']" --> ("metadata.annotations", "a[b]c", true)
-//  - "metadata.labels['']"           --> ("metadata.labels", "", true)
-//  - "metadata.labels"               --> ("metadata.labels", "", false)
+//   - "metadata.annotations['myKey']" --> ("metadata.annotations", "myKey", true)
+//   - "metadata.annotations['a[b]c']" --> ("metadata.annotations", "a[b]c", true)
+//   - "metadata.labels[”]"           --> ("metadata.labels", "", true)
```

Comment:
this is actually wrong... did gofmt make this change?

Reply:
yes, it did.

Let me replace that example using double quotes `metadata.labels[”]"`

Reply:
but double quotes aren't valid syntax for fieldPath indexing... only single-quotes are :-/

Reply:
is there a way to indent or wrap this so gofmt leaves it alone?

Reply:
on second thought, this might actually be a bug worth reporting to go... indenting and tweaking list bullets is one thing... replacing characters inside godoc strings is much more invasive

Reply:
looks like we want these to be code blocks:

> A code block is a span of indented or blank lines not starting with a bullet list marker or numbered list marker. It is rendered as preformatted text (a <pre> block in HTML).

maybe dropping the leading list markers and putting blank lines between them would make gofmt leave these alone?